tensorflow/tools/api/golden/tensorflow.layers.pbtxt

```diff
@@ -0,0 +1,517 @@
+path: "tensorflow.layers"
+tf_module {
+  name: "tensorflow.python.layers.layers"
```

Comment:
So the test will fail if we move a file and keep it the same name in python. Is this intentional? It's definitely the conservative choice, so it's probably for the best.